src/main/java/pro/sky/telegrambotanimalshelter/controller/HumanDogController.java

```diff
@@ -18,9 +18,9 @@
 @RequestMapping("human-dog")
 public class HumanDogController {
 
-    private final HumanDogService humanDogService;
+    private final HumanDogServiceImpl humanDogService;
```

Comment:
```suggestion
    private final HumanDogService humanDogService;
```
инъекцию надо делать по интерфейсу, а не реализации